### ISSUE TITLE
Customize top sites 

### ISSUE THREAD

**Issue by oscar345** [mozilla-mobile/firefox-ios#7546]
### Why/User Benefit/User Problem
now we add new top websites and we cannot change a thing. there is now way to rearrange the sites, change their name, or change the icon. I would like to have the same options as on the desktop version. 

### Acceptance Criteria (how do I know when I’m done?)
when the users are able to rearrange top sites in the order they like, they could change the name, and are able to change the icon by linking to a picture on the web.

**Comment by yoasif** [comment]
Seems like a partial dupe of https://github.com/mozilla-mobile/firefox-ios/issues/5283

**Comment by isaacjara17** [comment]
> Seems like a partial dupe of #5283

Not exactly, because this issue asks for the ability to change the name and the icon of the top site, which would be highly appreciated. Also +1

**Comment by dadpolice** [comment]
What needs to happen in order to get these features implemented? This is basic functionality and yet somehow this request has sat here with zero progress for two years.

**Comment by dadpolice** [comment]
Is there some high-ranking executive at Mozilla whose family was murdered by browser shortcuts or something? There has to be some reason why these features have been requested over and over again for years, and not only do they still not exist, but there's not been a single solitary shred of information from the Firefox developers about it. It has to go behind sheer incompetence at this point. For the love of god, please do something.

**Comment by lmarceau** [comment]
Hello @dadpolice, thank you for your concerns. As you might know, we get a lot of feature requests and issues reports from users. Any issues and feature requests raised by users are important and considered as we want Firefox for iOS to be a great product. Not all issues and feature requests are enabled though, as our 1k+ issues opened can attest. It seems to me you're asking why it is that way, and why our team sometimes doesn't enable features or answer requests. Briefly, as with any software development team, we have a finite amount of time and resources. Even if we wanted, we cannot enable all 1k+ issues. We're lucky enough that being open source, sometimes we have great contributions from the community to help us move the product forward, but most of the time decisions needs to be made by the product team so the development team can focus on certain features. If anyone feels strongly about a certain feature, nothing is preventing the community from building it and adding it to the browser. Anyone can also fork the project and steer it into the direction they want. Some well known iOS browsers where started that way.

As for the communication part, we do want to communicate more with our users and with contributors on what we're currently working on. We know this is a place for improvement. Now would be a great time if you have any proposals on the matter as this is something that will be looked into in the coming months.

**Comment by dadpolice** [comment]
@lmarceau What I am trying to get across is the fact that Firefox for iOS, in its current state, is unfinished. This is not a feature request or a "nice to have." You should not have the ability to pin favorites to the home screen if you cannot then edit or rearrange those favorites. That's an amateur implementation. You get a C- for that in school. It's as if you forgot to finish this feature and then it accidentally got into production. And then you sit here and act like this is such a rock bottom low priority item that you won't even respond to the many, repeated requests over the years for you to fix it. We're talking THREE YEARS since the first person mentioned it, and not only has there been no movement on the request, but you haven't even ADDRESSED it. No one has said a word about why you can't seem to finish the home screen. It's just there, half broken, and you don't care, and presumably you wonder why more people aren't using your unfinished app. This isn't just bad communication, it's no communication, and that's just not acceptable.

"If anyone feels strongly about a certain feature, nothing is preventing the community from building it and adding it to the browser."

Nothing makes me lose respect for a project faster than statements like this. Developing Firefox is not my job. It's yours. If this were some random esoteric feature that I only I care about, sure, put the onus on me to get the ball rolling. But it's not. It's a fundamental functionality your browser is missing and which your users have been asking for for years.

Proper communication isn't just "sorry we're busy but you can do it yourself if you want." Proper communication is explaining why you've actively ignored these requests for three years and what, exactly, is holding you up from implementing these features.
